cmake/vcpkg_setup.cmake

```diff
@@ -0,0 +1,78 @@
+
+include_guard()
+
+if (ROUGHPY_NO_VCPKG OR "${CMAKE_TOOLCHAIN_FILE}" MATCHES "vcpkg.cmake")
+    # Do not use vcpkg, or vcpkg toolchain already set up
+    message(DEBUG "Not using vcpkg or toolchain already set")
+    return()
+endif()
+
+
+function(_setup_vcpkg _path)
+    find_package(Git REQUIRED)
+    message(STATUS "Cloning vcpkg into \"${_real_path}\"")
```

Comment:
The variable `_real_path` is not defined. Should this be `_path` instead?
For reference, in the original code in the root `CMakeLists.txt` this variable was called `_vcpkg_dir`.

Reply:
Yes, well spotted.